SingularityUI/app/templates/deployDetail/deployHeader.hbs

```diff
@@ -0,0 +1,40 @@
+{{! Part of deployBase }}
+<header class='detail-header'>
+    <div class="row">
+        <div class="col-md-8">
+            <h5>
+                <span>
+                    Request <a href="{{appRoot}}/request/{{ data.requestId }}">{{ data.requestId }}</a>
+                </span>
+            </h5>
+        </div>
+        {{! Right side, buttons }}
+        <div class="col-md-4 button-container">
+            <a class="btn btn-default" data-request-id="{{ data.id }}" data-action="viewObjectJSON">
+                JSON
+            </a>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-8">
+            <h3>
+                <span>
+                    {{ data.deployId }}
+                </span>
+                <span class="deploy-state" data-state="{{ data.deployResult.deployState }}">
+                    {{ data.deployResult.deployState }}
```

Comment:
This can occasionally be blank, let's see if we can get a PENDING or something similar here for a deploy that is still in progress